ibm_mq/tests/test_ibm_mq_unit.py

```diff
@@ -203,8 +203,11 @@ def test_ssl_connection_creation(instance):
 
 
 def test_ssl_check_normal_connection_before_ssl_connection(instance_ssl_dummy):
+    import logging
+
     import pymqi
 
+    logger = logging.getLogger(__file__)
```

Comment:
Not really a suggestion, but wanted to check my understanding: Is the python `logging` package being used here instead of the dedicated check instance logger because this is a unit test against the check? Since the the check's methods now expect a logger argument.

Reply:
good catch, yeah we don't type check `tests/` afaik & was easier to do this than creating a check instance just for the logger